### PR TITLE
Add draft for Jan newsletter (2022)

### LINKED ISSUE #2254: Collect newsletter items for 2022 - January
### Summary

* We publish (almost) monthly newsletter on TinyLetter: https://tinyletter.com/TuringWay/archive, most items in that comes from our community members
* Please use this issue to share any update from your project/org/community and work in The Turing Way or beyond that are relavnt for our community

### What needs to be done?

- [ ] Add a short blurb (2-3 sentences) with relevant links that could be added in the newsletter
- [ ] Please do mention the month when this announcement should be made

### Who can help?

* Anyone who has an update to share

---

### Updates

*Lorem ipsum dolor sit amet, consectetur adipiscing.*

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #2254 

* Add the newsletter draft and images

### List of changes proposed in this PR (pull-request)

* Draft for the newsletter
* Images for the draft

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?
- [ ] All contents are covered
- [ ] reviewer's name is added 

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

**Comment by malvikasharan** [comment]
The inclusive language test error is because of "Masters student" which is detected as master. I will ignore that for the newsletter.

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+Chaired by Anelda van der Walt and Malvika Sharan, this panel will feature insights from Batool Almarzouq, Bobby Shabangu, Camila Rangel-Smith, David Pérez-Suárez, Langa Khumalo and Yanina Bellini Saibene. 
+These researchers represent different communities with expertise and insights from working in multilingual research and data science projects (read their bios below). 
+We will also facilitate open discussions with attendees to learn about their perspectives on the topic via a shared document.
+Please register via [the Eventbrite page](https://www.eventbrite.co.uk/e/fireside-chat-gaps-opportunities-for-inclusive-multilingual-data-science-tickets-255972068347).
+
+### Collaboration Cafés
+
+We resumed *The Turing Way* Collaboration Cafés in January and had a chance to catch up with several of our community members.
+The next one is on **2 February (Today!), from 15:00 - 17:00 UTC** ([see in your timezone](https://arewemeetingyet.com/london/2022-02-02/15:00/)).
+Collaboration Cafes are collaborative spaces for you to connect with *The Turing Way* team, share your ideas and exchange feedback.
```
Comment (RaoOfPhysics) [inline communications/newsletters/newsletter_31_Jan2022.md:36]: “Cafés” for consistency.

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+Hosted every first and third Wednesday, these are also opportunities for us to onboard new members and share what we are working on.
+Save the dates for the next calls on 16 February and 2 March.
+Sign up on this shared HackMD: [hackmd.io/@turingway/collaboration-cafe](https://hackmd.io/@turingway/collaboration-cafe).
+
+### Weekly Coworking Calls
+
+We host a weekly Coworking Call **every week on Monday from 11:00 - 12:00** London time.
```
Comment (RaoOfPhysics) [inline communications/newsletters/newsletter_31_Jan2022.md:43]: `from 11:00 to 12:00`

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+
+# January Round-Up, next Fireside Chat and other updates from The Turing Way
+
+Hello Turing Way friends!
+
+We hope you had a good start to the year 2022! ⭐️
+
+It has been lovely to see many of you at our January’s Collaboration Café and we hope to connect with you in the coming weeks.
+Here are a few opportunities to engage with *The Turing Way* community:
+* 🍵 Join the Collaboration Café on 2 February (this is today!), from 15:00 - 17:00 London time (see [details](https://hackmd.io/@turingway/collaboration-cafe)).
```
Comment (RaoOfPhysics) [inline communications/newsletters/newsletter_31_Jan2022.md:11]: `from 15:00 to 17:00`

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+Chaired by Anelda van der Walt and Malvika Sharan, this panel will feature insights from Batool Almarzouq, Bobby Shabangu, Camila Rangel-Smith, David Pérez-Suárez, Langa Khumalo and Yanina Bellini Saibene. 
+These researchers represent different communities with expertise and insights from working in multilingual research and data science projects (read their bios below). 
+We will also facilitate open discussions with attendees to learn about their perspectives on the topic via a shared document.
+Please register via [the Eventbrite page](https://www.eventbrite.co.uk/e/fireside-chat-gaps-opportunities-for-inclusive-multilingual-data-science-tickets-255972068347).
+
+### Collaboration Cafés
+
+We resumed *The Turing Way* Collaboration Cafés in January and had a chance to catch up with several of our community members.
+The next one is on **2 February (Today!), from 15:00 - 17:00 UTC** ([see in your timezone](https://arewemeetingyet.com/london/2022-02-02/15:00/)).
```
Comment (RaoOfPhysics) [inline communications/newsletters/newsletter_31_Jan2022.md:35]: `from 15:00 to 17:00`

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+Therefore, Malvika will be hosting two short **workshops on Facilitating Collaborative Sessions** such as those organised in *The Turing Way* community.
+Please sign up by selecting one of the two sessions:
+* [Register for the Workshop-1](https://turing-uk.zoom.us/meeting/register/tJMscuqsrj4jEtboVuSPRv1OYlfdWt4zOXKa): **14 February, 17:00 to 18:00 UTC** ([see in your timezone](https://arewemeetingyet.com/london/2022-02-14/17:00/))
+* [Register for the Workshop-2](https://turing-uk.zoom.us/meeting/register/tJMvdu-hqTwrGdSnAzuBnjztYBi9_mv7bz7Q): **18 February, 10:00 - 11:00 UTC** ([see in your timezone](https://arewemeetingyet.com/london/2022-02-18/10:00/))
+
+## News from the community
+
+![Illustration showing two people saying Welcome!](images/2022-01-welcome-2.png)
+
+*The Turing Way Community, & Scriberia. (2021). Illustrations from the Turing Way book dashes. Zenodo. https://doi.org/10.5281/zenodo.3332807*
```
Comment (RaoOfPhysics) [inline communications/newsletters/newsletter_31_Jan2022.md:59]: Mention licence, possibly?

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 6]
```diff
@@ -0,0 +1,156 @@
...
+Both positions have a central role in laying the groundwork for inclusive co-creation and contribute to *The Turing Way* acting as a bi-directional conduit to implement best practices for data science.
+The **deadline for application is 6 February 2022**.
+Please see this [Twitter thread by Kirstie Whitaker](https://twitter.com/turingway/status/1486685227858767874?s=20&t=6Jqhf-4GJ58rD0LgwTCFWg).
+
+### Capturing Impact from Your Work
+
+![screenshot of Kisrtie’s Tweet saying - Hey folks! An end of year request please Have you used The Turing Way? For any purpose! Helped you work reproducibly? Inspired a project? Introduced you to someone cool? It would be amazing if you could reply so I can follow up & capture this impact.](images/2022-01-kwtweet-3.png)
+
+*[Kirstie’s Tweet](https://twitter.com/kirstie_j/status/1469060341606801410?s=20&t=6Jqhf-4GJ58rD0LgwTCFWg) invites The Turing Way users and contributors to share how this project has helped their work.*
```
Comment (RaoOfPhysics) [inline communications/newsletters/newsletter_31_Jan2022.md:74]: Strip tracking here too.

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 7]
```diff
@@ -0,0 +1,156 @@
...
+* Margaret Wanjiku added the [documentation and metadata](https://the-turing-way.netlify.app/reproducible-research/rdm/rdm-metadata.html) subchapter in the Research Data Management chapter.
+* Vicky Hellon revised and updated the [Open Access](https://the-turing-way.netlify.app/reproducible-research/open/open-access.html) publication subchapter.
+* Malvika ana Alejandro Coca added a simple guidance for chairing [Collaboration Café](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html#chairing-an-online-collaboration-cafe) and [Coworking Calls](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-weekly.html#chairing-coworking-calls).
+
+*Have an idea for collaborations or a chapter? [Connect with us](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md#get-in-touch). Please explore [issues]() and Pull Requests](https://github.com/alan-turing-institute/the-turing-way/pulls) to collaborate with others on ongoing work.*
```
Comment (RaoOfPhysics) [inline communications/newsletters/newsletter_31_Jan2022.md:113]: Missing link for issues: https://github.com/alan-turing-institute/the-turing-way/issues

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 8]
```diff
@@ -0,0 +1,156 @@
...
+
+## News and Announcements from Software Sustainability Institute
+
+### Collaboration Workshop 2022: Register and Submit abstract!
+
+[Collaborations Workshop 2022 (CW22)](https://software.ac.uk/cw22) will take place online from Monday 4 April - Thursday 7 April 2022.
+The Software Sustainability Institute’s Collaborations Workshop series brings together researchers, developers, innovators, managers, funders, publishers, policy-makers, leaders and educators to explore best practices and the future of research software. 
+The themes of the CW22 keynote presentations, discussion groups, mini-workshops, collaborative ideas, and hack day will be around the following areas of research software: Code Review, Ethics, Hybrid Working, and Software Sustainability.
```
[the remark this answers is not in the document]
Comment (RaoOfPhysics) [reply communications/newsletters/newsletter_31_Jan2022.md:121]: Also after “Working”?

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 9]
```diff
@@ -0,0 +1,156 @@
...
+
+![Carlos Martinez's headshot](images/2022-01-carlos-4.png)
+
+In his words:
+> We always advocate for software reuse and collaborative development of software. 
+> I love that we can do the same for software development guidelines: reuse content from the eScience guide and collaboratively develop with The Turing Way community!
+> I am a big advocate of improving software quality. 
+> I am really glad that the eScience center is collaborating with The Turing Way in providing guidelines and helping build better research software.
+
+Thanks to Carlos’ work at organisation level.
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:101]:
```suggestion
Thanks to Carlos’ work at the organisation level.
```

**Review by RaoOfPhysics — CHANGES_REQUESTED** [review]

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 10]
```diff
@@ -0,0 +1,156 @@
...
+He is a certified Software Carpentry instructor and is frequently involved in organising training events.
+As a member of *The Turing Way* core team, he supports researchers from eScience Center in sharing internally developed resources and recommended practices for the [Netherlands eScience Center guide](https://guide.esciencecenter.nl/#/) to *The Turing Way*.
+
+![Carlos Martinez's headshot](images/2022-01-carlos-4.png)
+
+In his words:
+> We always advocate for software reuse and collaborative development of software. 
+> I love that we can do the same for software development guidelines: reuse content from the eScience guide and collaboratively develop with The Turing Way community!
+> I am a big advocate of improving software quality. 
+> I am really glad that the eScience center is collaborating with The Turing Way in providing guidelines and helping build better research software.
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:99]:
```suggestion
> I am really glad that the eScience Center is collaborating with The Turing Way in providing guidelines and helping build better research software.
```

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 1 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:36]:
```suggestion
Collaboration Cafés are collaborative spaces for you to connect with *The Turing Way* team, share your ideas and exchange feedback.
```

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 2 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:43]:
```suggestion
We host a weekly Coworking Call **every week on Monday from 11:00 to 12:00** London time.
```

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 3 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:11]:
```suggestion
* 🍵 Join the Collaboration Café on 2 February (this is today!), from 15:00 to 17:00 London time (see [details](https://hackmd.io/@turingway/collaboration-cafe)).
```

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 5 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:59]:
```suggestion
*The Turing Way Community, & Scriberia. (2021). Illustrations from the Turing Way book dashes shared under the CC BY 4.0 License. Zenodo. https://doi.org/10.5281/zenodo.3332807*
```

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:101]:
```suggestion
Thanks to Carlos’ work at the organisation level.
```

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 11]
```diff
@@ -0,0 +1,156 @@
...
+
+### Jobs at Turing!
+
+Apply for **Community Manager roles** within the *AI for Multiple Long Term Conditions Research Support Facility*.
+The Community Manager for *Patient and Public Involvement and Engagement* will work with researchers to elevate the voices of people living with multiple long term conditions, their family members and carers: [apply here](https://cezanneondemand.intervieweb.it/turing/jobs/community_manager_patient_and_public_involvement_and_engagement_aim_rsf_19281). 
+The Open Collaboration Community Manager will work with Theme Lead with researchers to deliver scientific community building and training: [apply here](https://cezanneondemand.intervieweb.it/turing/jobs/community_manager_open_collaboration_aim_rsf_19278).
+Both positions have a central role in laying the groundwork for inclusive co-creation and contribute to *The Turing Way* acting as a bi-directional conduit to implement best practices for data science.
+The **deadline for application is 6 February 2022**.
+Please see this [Twitter thread by Kirstie Whitaker](https://twitter.com/turingway/status/1486685227858767874?s=20&t=6Jqhf-4GJ58rD0LgwTCFWg).
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:68]:
```suggestion
Please see this [Twitter thread by Kirstie Whitaker](https://twitter.com/turingway/status/1486685227858767874.
```

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 7 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:113]:
```suggestion
*Have an idea for collaborations or a chapter? [Connect with us](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md#get-in-touch). Please explore [issues](https://github.com/alan-turing-institute/the-turing-way/issues) and Pull Requests](https://github.com/alan-turing-institute/the-turing-way/pulls) to collaborate with others on ongoing work.*
```

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 12]
```diff
@@ -0,0 +1,156 @@
...
+* Vicky Hellon revised and updated the [Open Access](https://the-turing-way.netlify.app/reproducible-research/open/open-access.html) publication subchapter.
+* Malvika ana Alejandro Coca added a simple guidance for chairing [Collaboration Café](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-collabcafe.html#chairing-an-online-collaboration-cafe) and [Coworking Calls](https://the-turing-way.netlify.app/community-handbook/coworking/coworking-weekly.html#chairing-coworking-calls).
+
+*Have an idea for collaborations or a chapter? [Connect with us](https://github.com/alan-turing-institute/the-turing-way/blob/main/CONTRIBUTING.md#get-in-touch). Please explore [issues]() and Pull Requests](https://github.com/alan-turing-institute/the-turing-way/pulls) to collaborate with others on ongoing work.*
+
+## News and Announcements from Software Sustainability Institute
+
+### Collaboration Workshop 2022: Register and Submit abstract!
+
+[Collaborations Workshop 2022 (CW22)](https://software.ac.uk/cw22) will take place online from Monday 4 April - Thursday 7 April 2022.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:119]:
```suggestion
[Collaborations Workshop 2022 (CW22)](https://software.ac.uk/cw22) will take place online from Monday 4 April to Thursday 7 April 2022.
```

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 6 above)
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:74]:
```suggestion
*[Kirstie’s Tweet](https://twitter.com/kirstie_j/status/1469060341606801410) invites The Turing Way users and contributors to share how this project has helped their work.*
```

**Review by RaoOfPhysics — APPROVED** [review]

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:99]:
```suggestion
> I am really glad that the eScience Center is collaborating with The Turing Way in providing guidelines and helping build better research software.
```

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 13]
```diff
@@ -0,0 +1,156 @@
...
+22 new Software Sustainability Institute Fellows have joined the fellows' network in 2022.
+In addition, for the first time, four International Fellows were selected under the International Fellowship pilot! 
+Two of these fellows are **Sophia Batchelor and Bastian Greshake Tzovaras** who are *The Turing Way* contributors and researchers collaborating on the [AutSPACEs](https://github.com/alan-turing-institute/AutSPACEs) and [Open Humans](https://www.openhumans.org/) project. 
+Congrats to all the fellows! Read [more about them](https://www.software.ac.uk/news/announcing-2022-software-sustainability-institute-fellows).
+
+## Twitter
+
+![Screenshot of 4 tweets in a pciture.](images/2022-01-tweetcollage-6.png)
+
+**1** [Sara El-Gebali](https://twitter.com/yalahowy/status/1469362864259579910?s=20&t=6Jqhf-4GJ58rD0LgwTCFWg) tweeted a picture of her cat co-watching December’s fireside chat with her.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:137]:
```suggestion
**1** [Sara El-Gebali](https://twitter.com/yalahowy/status/1469362864259579910) tweeted a picture of her cat co-watching December’s fireside chat with her.
```

File: communications/newsletters/newsletter_31_Jan2022.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (malvikasharan) [reply communications/newsletters/newsletter_31_Jan2022.md:121]:
```suggestion
The themes of the CW22 keynote presentations, discussion groups, mini-workshops, collaborative ideas and hack day will be around the following areas of research software: Code Review, Ethics, Hybrid Working and Software Sustainability. 
```

**Comment by malvikasharan** [comment]
Thank you @RaoOfPhysics for the thorough review. I have integrated your suggestions.
@GeorgiaHCA if you have time, please have a look at the revised draft. I will meanwhile update this in the tinyletters.

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 14]
```diff
@@ -0,0 +1,156 @@
...
+We hope you had a good start to the year 2022! ⭐️
+
+It has been lovely to see many of you at our January’s Collaboration Café and we hope to connect with you in the coming weeks.
+Here are a few opportunities to engage with *The Turing Way* community:
+* 🍵 Join the Collaboration Café on 2 February (this is today!), from 15:00 - 17:00 London time (see [details](https://hackmd.io/@turingway/collaboration-cafe)).
+* 🔥 Attend the Fireside Chat on *Gaps and Opportunities for Inclusive Multilingual Data Science* - 11 February, 14:00 - 15:30 UTC ([register](https://www.eventbrite.co.uk/e/fireside-chat-gaps-opportunities-for-inclusive-multilingual-data-science-tickets-255972068347)).
+* 💼 Apply for the Community Manager positions at The Alan Turing Institute to work closely with *The Turing Way*.
+* 👋 Sign  up to attend a short workshop on Facilitating Inclusive Events online on 14 and 18 February. 
+
+Read on for details, updates on new chapters,  👇
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:16]:
```suggestion
Read on for details, updates on new chapters and other news from the project!
```

Comment (RaoOfPhysics) [reply communications/newsletters/newsletter_31_Jan2022.md]: Aaaaah, how’d I miss this!

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 15]
```diff
@@ -0,0 +1,156 @@
...
+Congrats to all the fellows! Read [more about them](https://www.software.ac.uk/news/announcing-2022-software-sustainability-institute-fellows).
+
+## Twitter
+
+![Screenshot of 4 tweets in a pciture.](images/2022-01-tweetcollage-6.png)
+
+**1** [Sara El-Gebali](https://twitter.com/yalahowy/status/1469362864259579910) tweeted a picture of her cat co-watching December’s fireside chat with her.
+**2** Cassandra Gould van Praag recently released a training video introducing version control with GitLab. See [her tweet](https://twitter.com/cassgvp/status/1484606620286038016?s=20&t=6Jqhf-4GJ58rD0LgwTCFWg) for links to her talk along with other training videos.
+**3** Data Stewardship Council shares The Turing Way illustration on FAIR in a [recent tweet](https://twitter.com/DeltaCouncil/status/1480689241612976131?s=20&t=6Jqhf-4GJ58rD0LgwTCFWg).
+**4** Katie McDonough recommended The Turing Way in [her tweet](https://twitter.com/khetiwe24/status/1484167069058973703?s=20&t=6Jqhf-4GJ58rD0LgwTCFWg) as part of an independent study on machine learning in Digital Humanities.
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:140]:
```suggestion
**2** Cassandra Gould van Praag recently released a training video introducing version control with GitLab. See [her tweet](https://twitter.com/cassgvp/status/1484606620286038016) for links to her talk along with other training videos.
**3** Data Stewardship Council shares The Turing Way illustration on FAIR in a [recent tweet](https://twitter.com/DeltaCouncil/status/1480689241612976131).
**4** Katie McDonough recommended The Turing Way in [her tweet](https://twitter.com/khetiwe24/status/1484167069058973703) as part of an independent study on machine learning in Digital Humanities.
```

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 16]
```diff
@@ -0,0 +1,156 @@
...
+
+It has been lovely to see many of you at our January’s Collaboration Café and we hope to connect with you in the coming weeks.
+Here are a few opportunities to engage with *The Turing Way* community:
+* 🍵 Join the Collaboration Café on 2 February (this is today!), from 15:00 to 17:00 London time (see [details](https://hackmd.io/@turingway/collaboration-cafe)).
+* 🔥 Attend the Fireside Chat on *Gaps and Opportunities for Inclusive Multilingual Data Science* - 11 February, 14:00 - 15:30 UTC ([register](https://www.eventbrite.co.uk/e/fireside-chat-gaps-opportunities-for-inclusive-multilingual-data-science-tickets-255972068347)).
+* 💼 Apply for the Community Manager positions at The Alan Turing Institute to work closely with *The Turing Way*.
+* 👋 Sign  up to attend a short workshop on Facilitating Inclusive Events online on 14 and 18 February. 
+
+Read on for details, updates on new chapters and other news from the project! 👇
+
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:17]:
```suggestion

_Thanks to Achintya Rao for reviewing an earlier draft of this newsletter! 🌻_

```

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 17]
```diff
@@ -0,0 +1,158 @@
...
+On 11 February 2022, 14:00 - 15:30 UTC ([in your timezone](https://arewemeetingyet.com/london/2022-02-11/14:00/ttw-fireside-chat)), the next fireside chat will take place on the topic: **Gaps & Opportunities for Inclusive, Multilingual Data Science**. 
+Chaired by Anelda van der Walt and Malvika Sharan, this panel will feature insights from Batool Almarzouq, Bobby Shabangu, Camila Rangel-Smith, David Pérez-Suárez, Langa Khumalo and Yanina Bellini Saibene. 
+These researchers represent different communities with expertise and insights from working in multilingual research and data science projects (read their bios below). 
+We will also facilitate open discussions with attendees to learn about their perspectives on the topic via a shared document.
+Please register via [the Eventbrite page](https://www.eventbrite.co.uk/e/fireside-chat-gaps-opportunities-for-inclusive-multilingual-data-science-tickets-255972068347).
+
+### Collaboration Cafés
+
+We resumed *The Turing Way* Collaboration Cafés in January and had a chance to catch up with several of our community members.
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:35]:
```suggestion
We had a chance to catch up with several of you at the *The Turing Way* Collaboration Cafés.
```

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 18]
```diff
@@ -0,0 +1,158 @@
...
+The next one is on **2 February (Today!), from 15:00 to 17:00 UTC** ([see in your timezone](https://arewemeetingyet.com/london/2022-02-02/15:00/)).
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:36]:
```suggestion
```

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 19]
```diff
@@ -0,0 +1,158 @@
...
+Collaboration Cafés are collaborative spaces for you to connect with *The Turing Way* team, share your ideas and exchange feedback.
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:37]:
```suggestion
Taking place every first and third Wednesday for 2 hours from 15:00 to 17:00 UTC ([see in your timezone](https://arewemeetingyet.com/london/2022-02-16/15:00/)), Collaboration Cafés are collaborative spaces for you to connect with *The Turing Way* team, share your ideas and exchange feedback.
```

File: communications/newsletters/newsletter_31_Jan2022.md [hunk 20]
```diff
@@ -0,0 +1,158 @@
...
+Hosted every first and third Wednesday, these are also opportunities for us to onboard new members and share what we are working on.
```
Comment (malvikasharan) [inline communications/newsletters/newsletter_31_Jan2022.md:38]:
```suggestion
These are also opportunities for us to onboard new members and share what we are working on.
```

**Pull request merged**